test/test_python_dispatch.py

```diff
@@ -1821,5 +1823,28 @@ def __torch_dispatch__(cls, func, types, args, kwargs):
             e = LayoutDefaultReturn(torch.randn(4, 2), use_wrapper_subclass)
             self.assertEqual(e.layout, torch.strided)
 
+    def test_trace_offset(self):
+        class MockDispatchMode(TorchDispatchMode):
+            def __init__(self, traceback):
+                self.traceback = traceback
+
+            def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+                self.traceback.extend(traceback.format_stack())
+
+        # If this test stops working because of a change in Python dispatch,
+        # please make sure to change the constant itself.
+        expected_offset = _PYTHON_DISPATCH_TRACE_OFFSET
+        tb = []
+        with MockDispatchMode(tb) as dispatch:
+            # Tests if the line calling the kernel launch is at the expected offset
+            # in the stack trace using the fact that the unique variable name will
+            # appear in it.
+            _long_and_unique_launch_name_ = torch.rand(3)
+            self.assertNotIn(
```

Comment:
but you are not checking that unique name is in the trace? (In the next PR you are doing it for cuda trace offset)

Reply:
Sorry, fixed it now. The test did not work with CrossRef, I tried to work around it but in the end I decided skipping it is best.